src/newlines.cpp

```diff
@@ -6623,7 +6628,8 @@ static void newlines_double_space_struct_enum_union(Chunk *open_brace)
 
    Chunk *pc = Chunk::NullChunkPtr;
 
-   if (open_brace != nullptr)
+   if (  open_brace != nullptr
+      && open_brace->IsNotNullChunk())
```

Comment:
As some comments above, this is only a temporary check to prevent possible SEGV. No need to check for ```IsNotNullChunk()```